### DISCUSSION TITLE
user-guide/plugins/zotero-scihub

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#293] [文档评论区]
# user-guide/plugins/zotero-scihub

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/plugins/zotero-scihub

**Comment by Armstrong-2** [comment]
为什么插件下好后在zotero7的编辑—设置里没有显示呢？

**Comment by amea13** [comment]
右键并没有Update Scihub PDF选项

**Reply by jiaojiaodubai** [reply]
这是另一个 Scihub 插件的功能，现在自动集成到官方的“查找可用的 PDF”里面了

**Comment by tongyanggggggg** [comment]
为什么scipdf下好后仍然没办法抓取PDF呢？依然只有网页，查找全文显示未找到文件

**Comment by Yuan3295** [comment]
zotero7查找全文时，一直弹出Science Directd的人机验证，这是为什么呢？

**Reply by marshallfan** [reply]
我也遇到一样的情况

**Comment by whubao** [comment]
能不能在某一列显示doi

**Comment by shinecxw** [comment]
按照上面自定义 PDF Resolvers，Zotero已经被我改坏了，呜呜呜·
